### PR TITLE
Create mysample

### PR DESCRIPTION
**Pull request by devilrocker**

### CODE REVIEW

**Review by coolsanjayz — COMMENTED** [review]
add a _

**Comment by devilrocker** [comment]
> add a _

ok

**Review by coolsanjayz — CHANGES_REQUESTED** [review]
ADD A !

File: mysample [hunk 1]
```diff
@@ -0,0 +1 @@
+this is mine
```
Comment (devilrocker) [inline mysample:1]: ZXZ

**Pull request merged**